### PR TITLE
relocate toggle arrow button

### PR DESCRIPTION
**Pull request by GretaD**
ref #[2110](https://github.com/nextcloud/mail/issues/2110#issuecomment-548906027)

### CODE REVIEW

File: src/components/AppDetailsToggle.vue [hunk 1]
```diff
@@ -39,11 +39,10 @@ export default {
 	left: 0;
 	width: 44px;
 	height: 44px;
-	z-index: 149;
-	background-color: var(--color-main-background-darker);
+	z-index: 1051;
```
Comment (ChristophWurst) [inline src/components/AppDetailsToggle.vue]: 🧐🧐🧐

File: src/components/AppDetailsToggle.vue [hunk 2]
```diff
@@ -39,11 +39,10 @@ export default {
...
 	cursor: pointer;
-	opacity: 0.6;
 	transform: rotate(180deg);
-	margin-top: 44px; /* under the show navigation button */
+	background-color: white;
```
Comment (ChristophWurst) [inline src/components/AppDetailsToggle.vue]: what about dark theme? :)

Comment (GretaD) [reply src/components/AppDetailsToggle.vue]: Thats a very good question. Working on it

File: src/components/AppDetailsToggle.vue (hunk 1 above)
Comment (GretaD) [reply src/components/AppDetailsToggle.vue]: you dont like my 1051 index number? I can make it 2000 ;)

**Comment by jancborchardt** [comment]
Thanks for picking it up! :) There is the same issue in Contacts as well, mind fixing that the same way?
contacts back and menu

**Comment by ChristophWurst** [comment]
> Thanks for picking it up! :) There is the same issue in Contacts as well, mind fixing that the same way?

Mind submitting a review for this one? :)

**Comment by jancborchardt** [comment]
@GretaD commits should be squashed into 1, otherwise good! :)

**Pull request merged**